third_party/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation - All rights reserved
+# Licensed under the MIT License
+
+
+SB_MICRO_PATH="${SB_MICRO_PATH:-/usr/local}"
+
+# Build cutlass.
+if [ -f cutlass/CMakeLists.txt ]; then
```

Comment:
could we use `Makefile`? since many dependencies do not use cmake
`Makefile` could provide one target for each dependency, so it's easy to support building selected dependencies

Reply:
Since the building of thirdparty may vary a lot, I just used the most general way that using shell script. Each thirdparty project will have its own building script.

Reply:
Changed to Makefile.